src/libstrongswan/plugins/botan/botan_x25519.c

```diff
@@ -155,7 +155,7 @@ diffie_hellman_t *botan_x25519_create(diffie_hellman_group_t group)
 		},
 	);
 
-	if (botan_rng_init(&rng, "user"))
+	if (!botan_get_rng(&rng, RNG_TRUE))
```

Comment:
Note that `"user"` is usually mapped to `RNG_STRONG`.

Reply:
Missed that one.